src/librustc/session/config.rs

```diff
@@ -879,20 +911,20 @@ pub fn rustc_optgroups() -> Vec<RustcOptGroup> {
                                 located",
                  "NAME=PATH"),
         opt::opt("", "sysroot", "Override the system root", "PATH"),
-        opt::multi("Z", "", "Set internal debugging options", "FLAG"),
-        opt::opt_u("", "error-format", "How errors and other messages are produced", "human|json"),
```

Comment:
Or perhaps what @pnkfelix meant is that, more generally, just writing "opt" suggests stable? IOW, maybe we do want to change the name of `opt` etc to be `opt_s`, so that you are always explicitly stating the stability? (Might be more obvious this way)